dialects/linux/Makefile

```diff
@@ -18,7 +18,7 @@ CDEF=
 CDEFS=  ${CDEF} ${CFGF}
 DEP=	${CFGD} ${CFGDN}
 INCL=	${DINC}
-CFLAGS=	${CDEFS} ${INCL} ${DEP} ${DEBUG}
+CFLAGS=	-Wall ${CDEFS} ${INCL} ${DEP} ${DEBUG}
```

Comment:
Why not put this in the (currently empty) `CDEF` assignment below, which propagates to here anyway?

Reply:
I guess the original author introduced CDEF for allowing a user to add extra definition flags (-D) from the command line of make. So I didn't put '-Wall' to CDEF.  Just guessing.

Reply:
The way those variables are handled right now they always override anything provided on the command line. Would have to be `CDEFS?=` to allow overriding by the user.